eva-accession-ws/src/main/java/uk/ac/ebi/eva/accession/ws/rest/SubmittedVariantsRestController.java

```diff
@@ -93,5 +107,60 @@ public SubmittedVariantsRestController(
                         service.getLastInactive(identifier),
                         SubmittedVariant::new));
     }
-}
 
+    @GetMapping(value = "/beacon/query", produces = "application/json")
```

Comment:
do we have to make the endpoints so strange? can't this just be "/exists"?

Reply:
This is because of the API specification: see [here](https://github.com/ga4gh-beacon/beacon-elixir/blob/master/elixir_beacon/src/main/java/org/ega_archive/elixirbeacon/controller/ElixirBeaconController.java#L38)